fsm.go

```diff
@@ -72,7 +79,7 @@ func (r *Raft) runFSM() {
 	batchingFSM, batchingEnabled := r.fsm.(BatchingFSM)
 	configStore, configStoreEnabled := r.fsm.(ConfigurationStore)
 
-	commitSingle := func(req *commitTuple) {
+	applySingle := func(req *commitTuple) {
```

Comment:
I change this function name (and `applyBatch` below) because the operation it is performing is an Apply, not a commit.